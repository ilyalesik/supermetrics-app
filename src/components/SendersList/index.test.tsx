import React from "react";
import renderer from "react-test-renderer";
import { SendersList } from "./index";

it("renders SendersList", () => {
  const tree = renderer
    .create(
      <SendersList
        items={[
          {
            from_id: "user_1",
            from_name: "Filomena Cort",
            count: 2,
          },
          {
            from_id: "user_3",
            from_name: "Filomena Cort 2",
            count: 3,
          },
        ]}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
