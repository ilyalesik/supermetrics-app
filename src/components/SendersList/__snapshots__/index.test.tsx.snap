// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SendersList 1`] = `
<ul
  className="senders-lists"
>
  <a
    className="senders-lists-link"
    href="/dashboard/user_1"
    onClick={[Function]}
    to={null}
  >
    <li
      className="sender-list-item"
    >
      <p
        className="sender-list-item-title"
      >
        Filomena Cort
      </p>
      <div
        className="sender-list-item-count"
      >
        2
      </div>
    </li>
  </a>
  <a
    className="senders-lists-link"
    href="/dashboard/user_3"
    onClick={[Function]}
    to={null}
  >
    <li
      className="sender-list-item"
    >
      <p
        className="sender-list-item-title"
      >
        Filomena Cort 2
      </p>
      <div
        className="sender-list-item-count"
      >
        3
      </div>
    </li>
  </a>
</ul>
`;
